frontend/src/components/GameCard.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-container>
+    <v-card class="flex">
+        <v-card-title>{{gameTitle}}</v-card-title>
+        <v-row class="ma-2">
+            <v-col cols="6">
+                <v-img width="200" :src="image"></v-img>       
+                <v-card-actions class="center">
+                    <span class="grey--text text--lighten-2 text-caption">
+                        ({{ rating }})
+                    </span>
+                    <v-rating readonly v-model="rating" half-increments color="amber"></v-rating>
+                </v-card-actions>     
+            </v-col>
+            <v-col cols="6" class="flex">
```

Comment:
For mobile, we will probably want the overview to flex to the bottom. This can be done by setting the cols breakpoints. For example, `xs=12 md=6` . That way everything md and up will be the 6 cols and small to medium will be in one column and will flex below.

Reply:
Thanks!